### ISSUE TITLE
Both training loss and validation loss are always zero

### ISSUE THREAD

**Issue by JayCHaos** [JayCHaos/HSFCN#1]
I found that both training loss and validation loss are zero, and never changing. So I guess the network can never learn any positive features, although it did produce some results if using the trained model for prediction. The follow is the screen-shot of tensorboard results.
![tensorboard graphs](https://cloud.githubusercontent.com/assets/22782935/24856573/c8125b88-1e17-11e7-8d90-0201d9a2b111.png)
